src/app/[locale]/(profile)/profile/page.tsx

```
@@ -0,0 +1,9 @@
+import ProfileForm from '../_components/profile-form'
+import { getProfileData } from '../_actions/profile.actions';
```

ESLint

[error] Missing semicolon.

(semi)